lib/gds_api/test_helpers/content_register.rb

```diff
@@ -19,6 +19,11 @@ def stub_content_register_entries(format, entries)
           to_return(body: entries.to_json, status: 200)
       end
 
+      def content_register_isnt_available
+        stub_request(:any, /#{CONTENT_REGISTER_ENDPOINT}\/.*/)
+          .to_return(status: 503)
```

Comment:
The other methods in this module are split across lines with the `.` at the end of the first line - could you make this one consistent with that style?